MvvmCross/MvvmCross.csproj

```diff
@@ -74,6 +74,10 @@
     <Compile Include="Platform\Xamarin-common\**\*.cs" />
     <PackageReference Include="Tizen.NET" Version="4.0.0" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <None Include="..\readme.txt" pack="true" PackagePath="." />
```

Comment:
@martijn00 is there a reason why the readme is up a folder level? is this just so it doesn't get included as source code?

Reply:
I thought to add it as solution item, so it could work for every project. I could move it into the project itself.

Reply:
Given that it has a specific list of files I think move to project and then we may look at having an additional readme for mvvmcross.forms